### DISCUSSION TITLE
Player Claim Flag Customization for "interact-block-secondary"

### DISCUSSION THREAD

**Discussion by g-g-glitch** [Stardust-Labs-MC/StardustMC#50] [Ideas]
Requesting the `interact-block-secondary` flag be added to the list of player customizable flags for GriefDefender claims. This is useful for farms and mini-games to allow others the ability to use levers within the claim.

**Comment by aisukuma** [comment]
Added as a flag for `user` on the `public` section. This flag allows players to interact with levers, anvils, crafting tables, and grindstones.
